lib/mix/lib/mix/tasks/test.coverage.ex

```diff
@@ -369,6 +377,9 @@ defmodule Mix.Tasks.Test.Coverage do
   defp color(percentage, threshold) when percentage >= threshold, do: :green
   defp color(_, _), do: :red
 
+  defp format_number(number, length) when is_integer(number),
```

Comment:
this is okay? By default it's an integer which fails the formatting call. I wasn't sure what the convention was for converting to float or calling a function with changed params.